src/test/java/software/amazon/kinesis/connectors/flink/testutils/FakeKinesisFanOutBehavioursFactory.java

```diff
@@ -192,6 +196,12 @@ private SubscriptionErrorKinesisV2(final Throwable...throwables) {
 		@Override
 		void sendEvents(Subscriber<? super SubscribeToShardEventStream> subscriber) {
 			sendEventBatch(subscriber);
+			try {
```

Comment:
Seems like these changes were not in https://github.com/awslabs/amazon-kinesis-connector-flink/pull/18/commits
Should this be removed, or is there plan to make the same change on master?

Reply:
This was committed for v2.0.2 directly to fix a flaky test: https://github.com/awslabs/amazon-kinesis-connector-flink/commit/2c192fee24102c5e7b49eabadd79399f1ba0c373